### DISCUSSION TITLE
DHCP IPv6

### DISCUSSION THREAD

**Discussion by q7nm** [AdguardTeam/AdGuardHome#4179] [Q&A] [answered]
I setup AdGuardHome DHCP server on OpenWrt 21.02.1, but have not IPv6 addresses on clients. Is it not supported?

**Comment by ainar-g** [comment, marked as answer]
Hello.  Please elaborate on your setup.

Also, you can enable [verbose logging][1] and inspect the resulting logs for DHCP messages.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Reply by q7nm** [reply]
> Hello. Please elaborate on your setup.
> 
> Also, you can enable [verbose logging](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) and inspect the resulting logs for DHCP messages.

Hello! Today I reflashed OpenWrt, IPv6 DHCP worked, thanks for the answer. I think the problem was in a bad system configuration.
